### DISCUSSION TITLE
401( Unauthorized) from us-central1-maven.pkg.dev server

### DISCUSSION THREAD

**Discussion by EugeneAksyonov** [varabyte/kobweb#476] [Q&A] [answered]
Dear KobWeb Maintainers,
  I was trying to run "clock" example and gradle fails to retrieve artifacts for FontAwesome due to restricted access to the repo -
`Could not GET 'https://us-central1-maven.pkg.dev/varabyte-repos/public/com/varabyte/kobwebx/silk-icons-fa/0.17.0/silk-icons-fa-0.17.0.pom'. Received status code 401 from server: Unauthorized`
Probably I missed, but I could not find any information on the the project that requires authorization to the repo. Can you tell me what are the policies?
My Best Wishes,
Eugene.

**Comment by bitspittle** [comment]
Uh looks like an issue with Cloud billing. Investigating!

**Comment by bitspittle** [comment, marked as answer]
@EugeneAksyonov please try now (or, over the next 5-60 minutes, in case it takes time for the system to propagate)

**Reply by EugeneAksyonov** [reply]
Thanks! Works now :-)
Glad you resolved it!

**Comment by bitspittle** [comment]
Phew! Glad you reported it!
